api/mesh/v1alpha1/mesh.proto

```diff
@@ -227,15 +227,15 @@ message DatadogTracingBackendConfig {
   // `backend` service that communicates with a couple of databases, you would
   // get service names like `backend_INBOUND`, `backend_OUTBOUND_db1`, and
   // `backend_OUTBOUND_db2` in Datadog. Default: false
-  bool splitService = 3;
+  google.protobuf.BoolValue splitService = 3;
 }
 
 message ZipkinTracingBackendConfig {
   // Address of Zipkin collector.
   string url = 1;
 
   // Generate 128bit traces. Default: false
-  bool traceId128bit = 2;
```

Comment:
Doesn't it break helpers?  `bool` is false by default, now it can be nil.

Reply:
we unpack it with `pointer.Deref` https://github.com/kumahq/kuma/pull/12895/files#diff-adcdddd3271aaed6ae6844e294a9d0674c1dd87bd0117557f5d75a7ff0c1999aR104